gamestonk_terminal/portfolio/portfolio_analysis/portfolio_model.py

```diff
@@ -50,6 +54,12 @@ def load_portfolio(
             axis=1,
         )
 
+    if country:
+        df["country"] = df.apply(
+            lambda row: yfinance_model.get_country(row.Ticker),
+            axis=1,
+        )
```

Comment:
If the Dataframe gets large, I imagine this will take a while.  I think a better approach would be to create a dictionary using all unique tickers.  Something like `{tick: get_country(tick) for tick in df.Ticker.unique()}` and then you can do df["Country"] = df["Ticker"].map(country_dictionary}.  This will limit the number of times you get the country.

Reply:
Also done!